sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/logging/DiagnosticTelemetryPipelineListener.java

```diff
@@ -101,7 +101,7 @@ public void onResponse(TelemetryPipelineRequest request, TelemetryPipelineRespon
     public void onException(TelemetryPipelineRequest request, String reason, Throwable throwable) {
         if (!NetworkFriendlyExceptions.logSpecialOneTimeFriendlyException(
             throwable, request.getUrl().toString(), friendlyExceptionThrown, logger)) {
-            if (logRetryableFailures) {
+            if (logOnException) {
```

Comment:
If `logException` is false, then `OperationLogger` won't record a failure.

Then, `AggregatingLogger`  won't record a warning:

https://github.com/Azure/azure-sdk-for-java/blob/31fa48f2f9c4be643cd55a119f91cb709893f989/sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/logging/OperationLogger.java#L58

Then, 

```
WARN  c.a.m.o.e.i.p.TelemetryPipeline - Sending telemetry to the ingestion service: Channel response timed out after 60000 milliseconds. (...) (future warnings will be aggregated and logged once every 5 minutes)
java.util.concurrent.TimeoutException: Channel response timed out after 60000 milliseconds.
        at com.azure.core.http.netty.implementation.ResponseTimeoutHandler.responseTimedOut(ResponseTimeoutHandler.java:58)
        at com.azure.core.http.netty.implementation.ResponseTimeoutHandler.lambda$handlerAdded$0(ResponseTimeoutHandler.java:45)
        at io.netty.util.concurrent.PromiseTask.runTask(PromiseTask.java:98)
        at io.netty.util.concurrent.ScheduledFutureTask.run(ScheduledFutureTask.java:153)
        at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:403)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.lang.Thread.run(Thread.java:748)
```

won't be displayed: https://github.com/Azure/azure-sdk-for-java/blob/31fa48f2f9c4be643cd55a119f91cb709893f989/sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/logging/AggregatingLogger.java#L75

And 
```
WARN  c.a.m.o.e.i.p.TelemetryPipeline - In the last 5 minutes, the following operation has failed 1 times (out of 30): Sending telemetry to the ingestion service:
 * finishConnect(..) failed: Connection refused: (1 times)
```
wont't be displayed: https://github.com/Azure/azure-sdk-for-java/blob/31fa48f2f9c4be643cd55a119f91cb709893f989/sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/logging/AggregatingLogger.java#L152